src/core/varinfo.jl

```diff
@@ -169,56 +171,50 @@ end
 
 # Main behaviour control of rand() depending on sampler type and if sampler inside
 rand(vi::VarInfo, vn::VarName, dist::Distribution, spl::Sampler, inside=true) = begin
```

Comment:
I see. we can probably remove `produce(log(1.0))` since it doesn't change particle weights.


Ref: https://github.com/yebai/Turing.jl/blob/3ac05a4e19ead1e48a4e003fed68b39cecc46234/src/samplers/pgibbs.jl#L70